docs/site/Tutorials.md

```diff
@@ -0,0 +1,41 @@
+---
+lang: en
+title: Tutorials
+keywords: LoopBack 4.0
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Tutorials.html
+---
+
+LoopBack 4 comes with the following tutorials:
+
+- **[hello-world](https://github.com/strongloop/loopback-next/tree/master/examples/hello-world)**:
+  Tutorial on setting up a simple hello-world application using LoopBack 4.
```

Comment:
Are we sure that hello-world is a tutorial? AFAICT, it does not come with any actual tutorial guiding readers through the steps needed to build such application. Should we perhaps move hello-world to Examples section?

Reply:
I thought it was an example also, but the title said tutorial. I vote yes to move it to Examples.